app/models/log.rb

```diff
@@ -205,10 +205,6 @@ def blank_invalid_non_setup_fields!
   end
 
   (1..8).each do |person_num|
-    define_method("retirement_age_for_person_#{person_num}") do
-      retirement_age_for_person(person_num)
-    end
-
     define_method("plural_gender_for_person_#{person_num}") do
```

Comment:
I don't believe so, but it's always impossible to tell for sure with rails since someone _can_ do something very strange like `obj.send(%w(retirement age for person 1).join("_"))`
but I've global searched retirement_age_for which now only gets hits in the old form json files and I can't think of any other way it might be getting called that isn't far fetched.
The tests do all pass as well, so you would hope that that covers things?

Reply:
Ah, rails... That implies we probably could remove it but it seems right to be wary.